kata-deploy/kata-cleanup.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-cri-o-kata-cleanup
+  namespace: kube-system
+spec:
+  selector:
+      matchLabels:
+        name: kubelet-cri-o-kata-cleanup
+  template:
+    metadata:
+        labels:
+          name: kubelet-cri-o-kata-cleanup
+    spec:
+      serviceAccountName: kata-label-node
+      nodeSelector:
+          container-runtime: cri-o
+          kata-runtime: cleanup
+      containers:
+      - name: kube-kata-cleanup
+        image: egernst/kata-deploy
+        imagePullPolicy: Always
+        command: [ "sh", "-c" ]
+        args:
+        - kubectl label node $NODE_NAME container-runtime- kata-runtime-;
+          systemctl daemon-reload && systemctl restart crio && systemctl restart kubelet;
+          while true;
+          do sleep 36000; done;
```

Comment:
@jodh-intel DaemonSet pods should not exit. The controller will try to restart them in case they exit. That line was to workaround this behavior . Maybe `sleep infinity` ?

Reply:
`tail -f /dev/null` sleeps forever ;-)

Reply:
tailing /dev/null...

Reply:
ACK - will use tail -f!